VMBackup/main/parameterparser.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, protected_settings, public_settings, backup_logger):
                 backup_logger.log("settings received " + str(self.dynamicConfigsFromCRP), True)
                 for config in self.dynamicConfigsFromCRP:
                     if CommonVariables.key in config and CommonVariables.value in config:
-                        if(config[CommonVariables.key] in self.wellKnownSettingFlags):
+                        if(config[CommonVariables.key].lower() in self.wellKnownSettingFlags.lower()):
```

Comment:
why not keep all setting names as toLower in wellKnownSettingFlags instead of performing lower() operation for each key?

Reply:
The setting names while sending to BHS should be in camel case format like isSnapshotTtlEnabled.

Reply:
Then line 115 is going to have a problem because we are picking key value as config["key"] which will not match as it is not lowercase?